docs/en/sql-reference/functions/url-functions.md

```diff
@@ -48,10 +48,8 @@ clickhouse.com
 
 **Returned values**
 
-- Host name. If ClickHouse can parse the input string as a URL.
-- Empty string. If ClickHouse can’t parse the input string as a URL.
-
-Type: `String`.
+- Host name. If ClickHouse can parse the input string as a URL. [String](../data-types/string.md).
```

Comment:
--> single point

`Host name if ClickHouse can parse the input string as a URL, otherwise an empty string. [String](../data-types/string.md).`